### PR TITLE
Add base folder for plugin documentation

### PR DESCRIPTION
**Pull request by six42**
This PR has additional documentation:

1) Add base folder for plugin documentation as discussed in #685
 
 2) Add information about plugins and contacts to the standard page footer
On fitnesse.org the information about plugins is eye catching. Once FitNesse is installed on your own system no eye catching link is provided.
By adding a link to the footer this should advertise the plugins and hopefully more people will provide plugins.
For new users of FitNesse it is also important that they don't feel left alone and have an easy way to get help. The contact link in the footer should do the job.

### CODE REVIEW

**Comment by amolenaar** [comment]
I like the emphasis on plugins in the page footer.

How to make the plugins publish the documentation?

**Comment by six42** [comment]
At the beginning by giving manual instructions to the plugin users. Like: extract documentation.zip at FitnesseRoot/PlugIns

In a second step this can be automated. Fitnesse is doing it itself. So most of the code should be there.
It is just waiting for a brave volunteer to make it generic and working for plugins.

File: FitNesseRoot/FitNesse/UserGuide/content.txt [hunk 1]
```diff
@@ -13,6 +13,7 @@
  * [[Working with the FitNesse wiki][>FitNesseWiki]]
  * [[Writing Acceptance Tests][>WritingAcceptanceTests]] (jump directly to [[Fit][>WritingAcceptanceTests.FitFramework]] or [[Slim][>WritingAcceptanceTests.SliM]])
  * [[Administering FitNesse][>AdministeringFitNesse]]
+ * [[Extend Fitnesse with Plugins][.PlugIns]]
```
Comment (amolenaar) [inline FitNesseRoot/FitNesse/UserGuide/content.txt]: I don't think the name `Extend FitNesse with Plugins` covers it.

Maybe `Installed plugins`?

File: FitNesseRoot/PlugIns/content.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+!1 Get more out of Fitnesse
```
Comment (amolenaar) [inline FitNesseRoot/PlugIns/content.txt]: FitNesse should be spelled "FitNesse". (there are some other typos like that)

File: FitNesseRoot/PlugIns/content.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ * Additional Wiki Markup commands 
+ * Fixture frameworks you can use 
+ * and more ...
+
+See [[All Available Plugins][http://fitnesse.org/PlugIns?getPage&FitnesseVersion=${FITNESSE_VERSION}]] for a list of all available extensions.
+
+Read [[Writing Plugins][FitNesse.UserGuide.AdministeringFitNesse.WritingPlugins]] to understand how to install them or write your ownn.
+
+
+Below is the list of all installed plugins in this system:
```
Comment (amolenaar) [inline FitNesseRoot/PlugIns/content.txt]: That is: the plugins that have provided documentation ;).

File: FitNesseRoot/PageFooter/content.txt [hunk 4]
```diff
@@ -1 +1 @@
-[[Front Page][.FrontPage]] | [[User Guide][.FitNesse.UserGuide]] | [[root][root]] (for global !-!path's-!, ''etc.'') | Press '?' for keyboard shortcuts [[(edit)][.PageFooter?edit]]
\ No newline at end of file
+[[Front Page][.FrontPage]] | [[User Guide][.FitNesse.UserGuide]] | [[root][root]] (for global !-!path's-!, ''etc.'') | Press '?' for keyboard shortcuts  | [[Extend Fitnesse with Plugins][.PlugIns]] | [[Contact][http://fitnesse.org/StayInformed]] | [[(edit)][.PageFooter?edit]]
```
Comment (amolenaar) [inline FitNesseRoot/PageFooter/content.txt]: Change to `Installed plugins` or something similar.

**Comment by amolenaar** [comment]
I'm doing some Sunday morning nit picking ;). There's no feature available that installs plugins once the big "FitNesse update documentation" thingy has run. I think that's okay. This is nice and simple and it provides an easy to use convention.

Thank you :)

File: FitNesseRoot/PlugIns/content.txt (hunk 3 above)
Comment (six42) [inline FitNesseRoot/PlugIns/content.txt]: So true. 
The hope is that once we have a common place for documentation defined that each plugin developer will add something. 
Last but not least you can control it and not allow a plugin to be added to FitNesse.org which has no documentation at this place.

**Comment by six42** [comment]
all updated

**Comment by amolenaar** [comment]
It's in :)

**Pull request merged**